### PR TITLE
build: policy-check should include assert tagger

### PR DESCRIPTION
**Pull request by tylerbutler**
The policy-check command should, in general, include all policy handlers. However, assert tagging is a bit unique because we only want to run it as part of releases. An earlier change excluded the handler, but we still need it to run in "non-fix mode" when policy-check is run, so that it can catch issues like duplicate tags.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "lerna": "lerna",
     "lint": "lerna run lint --no-sort --stream -- -- -- --color",
     "lint:fix": "lerna run lint:fix --no-sort --stream -- -- -- --color",
-    "policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -D assert-short-codes",
+    "policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json",
     "policy-check:asserts": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json -d assert-short-codes --fix",
```
Comment (anthony-murphy) [inline package.json:48]: is this line wrong as well? i don't know what the command line syntax for this is? is it documented somewhere?

Comment (alexvy86) [reply package.json:48]: Docs are at https://github.com/microsoft/FluidFramework/tree/main/build-tools/packages/build-cli#fluid-toolsbuild-cli. `-d assert-short-codes` seems correct, it's the opposite of `-D`.

File: packages/dds/tree/src/feature-libraries/forestRepairDataStore.ts [hunk 2]
```diff
@@ -158,31 +158,29 @@ export class ForestRepairDataStore implements RepairDataStore {
     ): ITreeCursorSynchronous[] {
         const parent = getDescendant(this.root, path);
         const sparseField = parent.children.get(field);
-        assert(sparseField !== undefined, ERR_MISSING_DATA);
+        assert(sparseField !== undefined, "No repair data found");
```
Comment (tylerbutler) [inline packages/dds/tree/src/feature-libraries/forestRepairDataStore.ts:161]: @yann-achard-MS FYI, the assert tagger doesn't like this pattern. ☹️  I just removed the constant as a workaround.

File: package.json (hunk 1 above)
Comment (tylerbutler) [reply package.json]: I replaced the shortcut flags with their full forms so it's a little more self-documenting.

**Review by alexvy86 — APPROVED** [review]

**Comment by tylerbutler** [comment]
Thanks for the review, @alexvy86! I've merged the latest main in case there were any more introduced issues in the meantime. I'll merge as soon as CI passes.

**Pull request merged**